app/views/cookies/index.html.erb

```diff
@@ -1,5 +1,6 @@
 <%= turbo_frame_tag :cookies_modal do %>
   <% if cookies[:cookies_accepted].nil? %>
-    <%= Subdomain.current.cookies_consent_ui.html_safe %>
+      <div class="cookies-consent__overlay"></div>
```

Comment:
@Ayon95 adding this stub here makes customizing the consent popup more difficult than it should be. 

What if the client doesn't like the grayed out background? In this case it will be difficult to override it. 

Can you add the same functionality but only keep it to the migration? (as in remove this change)